src/components/hmrc-print-link/hmrc-print-link.yaml

```
@@ -0,0 +1,19 @@
+params:
+  - name: id
+    type: string
+    required: false
+    description: id to use within the govuk button component.
+  - name: text
+    type: string
+    required: false
+    description: Text to use within the govuk button component.
+  - name: attributes
+    type: object
+    required: false
+    description: HTML attributes (for example data attributes) to add to the anchor tag.
```

sorry I didn't spot this before 😮‍💨 I'm going to approve anyway since I think no-one ever looks at params and probably wouldn't expect to for this, but if you do feel like it:

```suggestion
params: []
```